src/memory/RomWonderKid.hh

```diff
@@ -0,0 +1,22 @@
+#ifndef ROMWONDERKID_HH
+#define ROMWONDERKID_HH
+
+#include "RomBlocks.hh"
+
+namespace openmsx {
+
+class RomWonderKid final : public Rom16kBBlocks
+{
+public:
+	RomWonderKid(const DeviceConfig& config, Rom&& rom);
+
+	void reset(EmuTime time) override;
+        [[nodiscard]] byte peekMem(uint16_t address, EmuTime time) const override;
+        [[nodiscard]] byte readMem(uint16_t address, EmuTime time) override;
+	void writeMem(uint16_t address, byte value, EmuTime time) override;
```

Comment:
This uses a mix of "4 spaces" and "1 tab(?)" for indentation. Usually in openMSX we use tabs for indentation.
Same in the file `RomWonderKid.cc`, although there it does seem consistent but uses spaces instead of tabs.

I'm not opposed to switching to 4 tabs for indentation, that seems to be to most common convention nowadays. But then maybe we should do that for the whole openMSX code base?

And for sure within a single file we shouldn't use a mix.